tests/integration/bash_tests/run_from_perlmutter/test_ls_globus.bash

```diff
@@ -4,6 +4,9 @@ hpss_globus_endpoint="6c54cade-bde5-45c1-bdea-f4bd71dba2cc"
 hpss_path="globus://${hpss_globus_endpoint}/~/zstash_test/"
 cache=zstash # Set via `self.cache = "zstash"`
 
+# Start fresh by deleting token file:
+rm ~/.zstash_globus_tokens.json
```

Comment:
This is the fix for the issue noted in the [E3SM Unified 1.12.0rc2 error](https://github.com/E3SM-Project/zstash/discussions/391#discussioncomment-14766431)